src/encord_active/lib/metrics/io.py

```diff
@@ -0,0 +1,84 @@
+import importlib.util
+import inspect
+from pathlib import Path
+from typing import Callable, Optional, Union
+
+from encord_active.lib.metrics.execute import logger
+from encord_active.lib.metrics.metric import Metric, SimpleMetric
+
+
+def fill_metrics_meta_with_builtin_metrics(metrics_meta: dict):
```

Comment:
We always seem to be calling this with an empty dictionary. Perhaps at least make the parameter optional to avoid specifying that argument.

Reply:
done 👍